frontend/config-assistant/src/store/sessionStore.ts

```diff
@@ -157,6 +161,14 @@ export const useSessionStore = defineStore('commonStore', () => {
 
   const schemaAtCurrentPath: Ref<JsonSchema> = computed(() => schemaAtPath(currentPath.value));
 
+  const effectiveSchemaAtCurrentPath: Ref<EffectiveSchema> = computed(() =>
```

Comment:
Ah shit this doesn't work at deeper levels

Reply:
okay but what should we do about it then? Do you see changing this as within the scope of this PR?

Reply:
i just forgot to push